label_studio/projects/api.py

```diff
@@ -146,7 +146,18 @@ def perform_create(self, ser):
             raise LabelStudioDatabaseException('Database error during project creation. Try again.')
 
     def get(self, request, *args, **kwargs):
-        return super(ProjectListAPI, self).get(request, *args, **kwargs)
+        from projects.cache import cached_projects_get, cached_projects_set
+
+        user_id = self.request.user.id
+        response = cached_projects_get(user_id)
```

Comment:
Unfortunately, we cannot cache projects, they change intensively every second. We can cache only particular fields like `project.members` for 10-20 min on project cards. How often this cache will be refreshed?

Reply:
Sergei echoed the same response when I first brought this up. It sounded like you guys had other ideas which is why I stopped working this. However I think it's worth some more time exploring, which I will do on my down time.

I think the key to having this work is to focus on caching each counter. Each counter would have its own cache. They would be be updated/deleted on any model save and after a certain amount of time.

> How often this cache will be refreshed?

We can specify any amount of time. 5 minutes is the default. I also set the cache to be deleted *ANY* time a project is saved (for any reason). See ```label_studio/projects/models.py -> cached_projects_remove(self.created_by_id)```

@makseq